### PR TITLE
ns-api: added flashstart api

### PR DESCRIPTION
**Pull request by Tbaile**
https://trello.com/c/5gPBuC4u/137-flashstart-api

### CODE REVIEW

File: packages/ns-api/README.md [hunk 1]
```diff
@@ -2162,3 +2162,50 @@ Example response:
...
+      "enabled": false,
+      "username": "...",
+      "password": "...",
+      "zones": [
+         "..."
+      ],
+      "bypass": []
+   }
+}
+
```
Comment (gsanchietti) [inline packages/ns-api/README.md:2192]: Please add some values, it's easier to read
```suggestion
{
   "values": {
      "enabled": false,
      "username": "myuser",
      "password": "mypass",
      "zones": [
         "lan"
      ],
      "bypass": [
         "192.168.1.1"
      ]
   }
}

```

File: packages/ns-api/README.md [hunk 2]
```diff
@@ -2162,3 +2162,50 @@ Example response:
...
+```
+
+### set-config
+
+Sets configuration of Flashstart service, **BEWARE** this commits directly the uci changes due to the need to restart
+flashstart service. The automatic uci commit is made inside `dhcp` and `flashstart` config.
+
+```bash
+api-cli ns.flashstart set-config --data '{"enabled": true, "username": "...", "password": "...", "zones": [...], "bypass": [...]}'
```
Comment (gsanchietti) [inline packages/ns-api/README.md:2201]: See comment above.

File: packages/ns-api/files/ns.flashstart [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+                  if zone != ''],
+        'bypass': [bypass for bypass in e_uci.get('flashstart', 'global', 'bypass', list=True, default=[])
+                   if bypass != '']
+    }
+
+
+cmd = sys.argv[1]
+
+if cmd == 'list':
+    print(json.dumps({
```
Comment (gsanchietti) [inline packages/ns-api/files/ns.flashstart:31]: We need also an api to list the zones. It should return all zones except wan.

File: packages/ns-api/files/ns.flashstart.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "dpi-manager": {
```
Comment (gsanchietti) [inline packages/ns-api/files/ns.flashstart.json:2]:
```suggestion
  "flashstart-manager": {
```

**Review by gsanchietti — CHANGES_REQUESTED** [review]

File: packages/ns-api/files/ns.flashstart (hunk 3 above)
Comment (Tbaile) [reply packages/ns-api/files/ns.flashstart:31]: So it's based off firewall configuration?

**Review by gsanchietti — APPROVED** [review]

**Pull request merged**